doc/book/src/introduction.md

```diff
@@ -6,8 +6,6 @@ micro-containers, then distributes them to be run on an arbitrarily large
 cluster of test-runners, or locally on your machine. You might use
 Maelstrom to run your tests because:
 
-* It's easy. Maelstrom functions as a drop-in replacement for `cargo test`, so in
-  most cases, it just works.
```

Comment:
IMO, this shouldn't be the #1 reason to use your product. First and foremost, it should solve a problem of mine.

Reply:
Yeah, that's a good point to consider.